modules/atomic-widgets/style/style-render.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\Style;
+
+class Style_Render {
+	private array $styles;
+	private array $transformers;
+	private array $breakpoints;
+
+	public function __construct( array $styles, array $transformers, array $breakpoints ) {
+		$this->styles = $styles;
+		$this->transformers = $transformers;
+		$this->breakpoints = $breakpoints;
+	}
+
+	public function render(): string {
```

Comment:
I think the styles should be passed as a parameter to the `render`:

```PHP
$renderer = new Styles_Renderer( $transformers );

$css1 = $renderer->render( $styles1 );
$css2 = $renderer->render( $styles2 );
// ...
```

Reply:
Good idea, will do it (breakpoints would also be passed to the constructor)